couchdb/templates/statefulset.yaml

```diff
@@ -108,6 +112,40 @@ spec:
           - name: database-storage
             mountPath: /opt/couchdb-search/data
 {{- end }}
+{{- if .Values.telemetry.enabled }}
+        - name: telemetry
+          image: "{{ .Values.telemetryImage.repository }}:{{ .Values.telemetryImage.tag }}"
+          imagePullPolicy: {{ .Values.telemetryImage.pullPolicy }}
+          ports:
+            - name: telemetry
+              containerPort: 9984
+          args: ["--couchdb.uri=http://localhost:5984"]
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 9984
+          readinessProbe:
+            httpGet:
+              path: /metrics
+              port: 9984
+          resources:
+{{ toYaml .Values.telemetry.resources | indent 12 }}
+          env:
+            - name: DATABASES
+              value: {{ .Values.telemetry.databases }}
+{{- if not .Values.allowAdminParty }}
+            - name: "COUCHDB.USERNAME"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "couchdb.fullname" . }}
+                  key: adminUsername
+            - name: "COUCHDB.PASSWORD"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "couchdb.fullname" . }}
```

Comment:
This one should be something like `"couchdb.defaultsecret"`?

Reply:
good catch - fixed!